src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -131,31 +135,32 @@ export class FeedCardComponent implements OnInit {
 		}
 	}
 
-	private sanitizeAndEmbedURLs(links) {
-		links.forEach((link, i) => {
-			const videoid = links[i].match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
+	private sanitizeVideoURLs() {
+		this.feedItem.videos.forEach((video) => {
+			const videoid = video.match(/(?:https?:\/{2})?(?:w{3}\.)?youtu(?:be)?\.(?:com|be)(?:\/watch\?v=|\/)([^\s&]+)/);
```

Comment:
here there are different quality videos available (see in the url) like 480*480, 720*720, 240*240